### PR TITLE
Create / Update UI components per Figma spec

### PR DESCRIPTION
**Pull request by Noxsios**
Adds / modifies the following components:

- [x] Action Dialog / Error Dialog
- [x] Configure Deployment page redesign
- [x] Section Header

Subcomponents built:

- Drawer
- Tooltip POC (will be upgraded in upcoming PR)

### CODE REVIEW

**Comment by Noxsios** [comment]
Action Dialog

Screenshot 2022-12-05 at 11 47 45 AM

**Comment by Noxsios** [comment]
> Is this still draft?

Once all components listed in the description are created + pass cursory inspection, will mark as ready.

**Comment by Noxsios** [comment]
Screenshot 2022-12-07 at 12 15 16 AM

Progress updating configure page.  Note new section headers, optional action buttons, tooltips.

**Comment by Noxsios** [comment]
Some elements are not 1:1 with the figma spec, this is due to disrepencies to the current global theme settings which will be fixed in an upcoming PR.

File: src/ui/routes/initialize/configure/+page.svelte [hunk 1]
```diff
@@ -3,51 +3,135 @@
...
+			<div class="align-center">
+				<Typography variant="body2">{$pkgStore.zarfPackage.build?.timestamp}</Typography>
+			</div>
+			<div class="align-center"><Typography variant="caption">Signed by:</Typography></div>
+			<div class="align-center">
+				<Typography variant="body2" />
+			</div>
+		</div>
+
+		<!-- @Noxsios: TODO: SBOM section, requires a change in how we handle SBOM generation + interaction with the frontend
```
Comment (jeff-mccoy) [inline src/ui/routes/initialize/configure/+page.svelte:84]: Can you make this a todo comment vs comment w/code?

**Comment by jeff-mccoy** [comment]
Can you drop the latest screenshots for the bullet points addressed in this PR?

**Comment by Noxsios** [comment]
New `ErrNotFound` dialog:

https://user-images.githubusercontent.com/50058333/207982817-d01a181d-9af2-4766-82f4-1c923950eb3c.mov

**Comment by Noxsios** [comment]
Configure page redesign: 

> **NOTE**
> There are inconsistencies between typography variants shown in this recording and the design spec, these will be remedied in an upcoming theme update PR

https://user-images.githubusercontent.com/50058333/207983208-5a77fbab-5827-4c08-9e92-8a176226b6be.mov

File: src/ui/routes/initialize/configure/+page.svelte [hunk 2]
```diff
@@ -3,51 +3,127 @@
...
+	import AccordionGroup from '$lib/components/accordion-group.svelte';
 
-	import Icon from '$lib/components/icon.svelte';
-	import PackageDetails from '$lib/components/package-details-card.svelte';
-	import PackageComponent from '$lib/components/package-component-accordion.svelte';
+	import { SectionHeader, ComponentAccordion as PackageComponent } from '$lib/components/package';
 	import { pkgStore } from '$lib/store';
-	import { Button, Typography } from '@ui';
+	import { Button, Chip, Typography } from '@ui';
+	let showRaw: boolean = false;
```
Comment (jeff-mccoy) [inline src/ui/routes/initialize/configure/+page.svelte:11]: Can we use implicit typing here as we do throughout the rest of the codebase? `let showRaw = false;`

**Review by Madeline-UX — COMMENTED** [review]
Functionality looks good 

Areas that need to be addressed before merging
- draw should close on small screen sizes

Issues to be fixed ASAP - before next release
- Typography fixes

**Pull request closed without merging**